src/Header/RequestParser.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "./OperateData.h"
+
+//used to make request string to OperateData(JSON like) format.
+class RequestParser
+{
+public:
+    //singleton code convention method.
+    static RequestParser* Get();
+    //give request string return header
+    static OperateData ParseHeader(std::string request);
```

Comment:
Consider returning a unique_ptr of OperateData to avoid copying.

Reply:
Have changed to return unique_ptr.